cypress/integration/internal/change-user-permissions.spec.js

```diff
@@ -24,15 +24,15 @@ describe('change internal user permissions as B&D user', () => {
       cy.get('.govuk-list .govuk-link').click();
 
       describe('navigates to the user page', () => {
-        cy.url().should('include', '/status');
+        cy.url().should('contain', '/user/');
         cy.contains('Internal').should('be.visible');
         cy.get('.govuk-heading-l').eq(0).should('contain', EMAIL_ADDRESS);
         cy.get('.govuk-heading-l').eq(1).should('have.text', 'Set permissions');
       });
       describe('changes the user permissions and navigates to the success page', () => {
         cy.get('#permission-4').check();
         cy.get('form > .govuk-button').click();
-        cy.url().should('include', '/update-permissions/success');
+        cy.url().should('contain', '/user/');
```

Comment:
Why did you change this one?

Reply:
you asked to keep user in your previous comment ? So i replaced. :)